### PR TITLE
Update Cold Archive guide for Beta release

### PR DESCRIPTION
**Pull request by jlutran**

### CODE REVIEW

**Review by Pofilo — APPROVED** [review]

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.de-de.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
 ---
-title: Cold Archive - Getting started with Cold Archive (Alpha) (EN)
+title: Cold Archive - Getting started with Cold Archive (Beta)
 slug: s3/getting-started-with-cold-archive
-routes:
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.de-de.md:4]: Please re-integrate the canonical link on the subs where it's deleted, replacing the link with https://docs.ovh.com/gb/en/storage/object-storage/s3/getting-started-with-cold-archive/

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.de-de.md [hunk 2]
```diff
@@ -18,13 +16,9 @@ Restoration can take some time as it needs to be read on tapes.
...
 **This guide explains how to set up storage on tapes with Cold Archive.**
 
-> [!primary]
->
-> During the the Alpha period, tapes are not used.
-
 ## Requirements
 
-- [Getting started with AWS CLI](https://docs.ovh.com/de/storage/s3/getting-started-with-s3/#using-the-aws-cli)
+- [Getting started with AWS CLI](https://docs.ovh.com/gb/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.de-de.md]:
```suggestion
- [Getting started with AWS CLI](https://docs.ovh.com/de/storage/object-storage/s3/getting-started-with-object-storage/#using-aws-cli)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.en-asia.md [hunk 3]
```diff
@@ -16,13 +16,9 @@ Restoration can take some time as it needs to be read on tapes.
...
 **This guide explains how to set up storage on tapes with Cold Archive.**
 
-> [!primary]
->
-> During the the Alpha period, tapes are not used.
-
 ## Requirements
 
-- [Getting started with AWS CLI](https://docs.ovh.com/asia/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
+- [Getting started with AWS CLI](https://docs.ovh.com/gb/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.en-asia.md]:
```suggestion
- [Getting started with AWS CLI](https://docs.ovh.com/asia/en/storage/object-storage/s3/getting-started-with-object-storage/#using-aws-cli)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.en-au.md [hunk 4]
```diff
@@ -16,13 +16,9 @@ Restoration can take some time as it needs to be read on tapes.
...
 **This guide explains how to set up storage on tapes with Cold Archive.**
 
-> [!primary]
->
-> During the the Alpha period, tapes are not used.
-
 ## Requirements
 
-- [Getting started with AWS CLI](https://docs.ovh.com/au/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
+- [Getting started with AWS CLI](https://docs.ovh.com/gb/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.en-au.md]:
```suggestion
- [Getting started with AWS CLI](https://docs.ovh.com/au/en/storage/object-storage/s3/getting-started-with-object-storage/#using-aws-cli)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.en-ca.md [hunk 5]
```diff
@@ -16,13 +16,9 @@ Restoration can take some time as it needs to be read on tapes.
...
 **This guide explains how to set up storage on tapes with Cold Archive.**
 
-> [!primary]
->
-> During the the Alpha period, tapes are not used.
-
 ## Requirements
 
-- [Getting started with AWS CLI](https://docs.ovh.com/ca/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
+- [Getting started with AWS CLI](https://docs.ovh.com/gb/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.en-ca.md]:
```suggestion
- [Getting started with AWS CLI](https://docs.ovh.com/ca/en/storage/object-storage/s3/getting-started-with-object-storage/#using-aws-cli)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.en-gb.md [hunk 6]
```diff
@@ -16,10 +16,6 @@ Restoration can take some time as it needs to be read on tapes.
 
 **This guide explains how to set up storage on tapes with Cold Archive.**
 
-> [!primary]
->
-> During the the Alpha period, tapes are not used.
-
 ## Requirements
 
 - [Getting started with AWS CLI](https://docs.ovh.com/gb/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.en-gb.md]:
```suggestion
- [Getting started with AWS CLI](https://docs.ovh.com/gb/en/storage/object-storage/s3/getting-started-with-object-storage/#using-aws-cli)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.en-ie.md [hunk 7]
```diff
@@ -16,13 +16,9 @@ Restoration can take some time as it needs to be read on tapes.
...
 **This guide explains how to set up storage on tapes with Cold Archive.**
 
-> [!primary]
->
-> During the the Alpha period, tapes are not used.
-
 ## Requirements
 
-- [Getting started with AWS CLI](https://docs.ovh.com/ie/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
+- [Getting started with AWS CLI](https://docs.ovh.com/gb/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.en-ie.md]:
```suggestion
- [Getting started with AWS CLI](https://docs.ovh.com/ie/en/storage/object-storage/s3/getting-started-with-object-storage/#using-aws-cli)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.en-sg.md [hunk 8]
```diff
@@ -16,13 +16,9 @@ Restoration can take some time as it needs to be read on tapes.
...
 **This guide explains how to set up storage on tapes with Cold Archive.**
 
-> [!primary]
->
-> During the the Alpha period, tapes are not used.
-
 ## Requirements
 
-- [Getting started with AWS CLI](https://docs.ovh.com/sg/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
+- [Getting started with AWS CLI](https://docs.ovh.com/gb/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.en-sg.md]:
```suggestion
- [Getting started with AWS CLI](https://docs.ovh.com/sg/en/storage/object-storage/s3/getting-started-with-object-storage/#using-aws-cli)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.en-us.md [hunk 9]
```diff
@@ -16,13 +16,9 @@ Restoration can take some time as it needs to be read on tapes.
...
 **This guide explains how to set up storage on tapes with Cold Archive.**
 
-> [!primary]
->
-> During the the Alpha period, tapes are not used.
-
 ## Requirements
 
-- [Getting started with AWS CLI](https://docs.ovh.com/us/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
+- [Getting started with AWS CLI](https://docs.ovh.com/gb/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.en-us.md]:
```suggestion
- [Getting started with AWS CLI](https://docs.ovh.com/us/en/storage/object-storage/s3/getting-started-with-object-storage/#using-aws-cli)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.es-es.md [hunk 10]
```diff
@@ -18,13 +16,9 @@ Restoration can take some time as it needs to be read on tapes.
...
 **This guide explains how to set up storage on tapes with Cold Archive.**
 
-> [!primary]
->
-> During the the Alpha period, tapes are not used.
-
 ## Requirements
 
-- [Getting started with AWS CLI](https://docs.ovh.com/es/storage/s3/getting-started-with-s3/#using-the-aws-cli)
+- [Getting started with AWS CLI](https://docs.ovh.com/gb/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.es-es.md]:
```suggestion
- [Getting started with AWS CLI](https://docs.ovh.com/es/storage/object-storage/s3/getting-started-with-object-storage/#using-aws-cli)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.es-es.md [hunk 11]
```diff
@@ -1,14 +1,12 @@
 ---
-title: Cold Archive - Getting started with Cold Archive (Alpha) (EN)
+title: Cold Archive - Getting started with Cold Archive (Beta)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.es-es.md]:
```suggestion
title: Cold Archive - Getting started with Cold Archive (Beta) (EN)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.de-de.md [hunk 12]
```diff
@@ -1,14 +1,12 @@
 ---
-title: Cold Archive - Getting started with Cold Archive (Alpha) (EN)
+title: Cold Archive - Getting started with Cold Archive (Beta)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.de-de.md]:
```suggestion
title: Cold Archive - Getting started with Cold Archive (Beta) (EN)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.es-us.md [hunk 13]
```diff
@@ -1,14 +1,12 @@
 ---
-title: Cold Archive - Getting started with Cold Archive (Alpha) (EN)
+title: Cold Archive - Getting started with Cold Archive (Beta)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.es-us.md]:
```suggestion
title: Cold Archive - Getting started with Cold Archive (Beta) (EN)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.es-us.md [hunk 14]
```diff
@@ -18,13 +16,9 @@ Restoration can take some time as it needs to be read on tapes.
...
 **This guide explains how to set up storage on tapes with Cold Archive.**
 
-> [!primary]
->
-> During the the Alpha period, tapes are not used.
-
 ## Requirements
 
-- [Getting started with AWS CLI](https://docs.ovh.com/us/es/storage/s3/getting-started-with-s3/#using-the-aws-cli)
+- [Getting started with AWS CLI](https://docs.ovh.com/gb/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.es-us.md]:
```suggestion
- [Getting started with AWS CLI](https://docs.ovh.com/us/es/storage/object-storage/s3/getting-started-with-object-storage/#using-aws-cli)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.fr-ca.md [hunk 15]
```diff
@@ -18,13 +18,9 @@ Restoration can take some time as it needs to be read on tapes.
...
 **This guide explains how to set up storage on tapes with Cold Archive.**
 
-> [!primary]
->
-> During the the Alpha period, tapes are not used.
-
 ## Requirements
 
-- [Getting started with AWS CLI](https://docs.ovh.com/ca/fr/storage/s3/debuter-avec-s3/#utilisation-de-aws-cli)
+- [Getting started with AWS CLI](https://docs.ovh.com/fr/storage/s3/debuter-avec-s3/#utilisation-de-aws-cli)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.fr-ca.md]:
```suggestion
- [Getting started with AWS CLI](https://docs.ovh.com/ca/fr/storage/object-storage/s3/getting-started-with-object-storage/#utilisation-de-aws-cli)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.fr-fr.md [hunk 16]
```diff
@@ -18,10 +18,6 @@ Restoration can take some time as it needs to be read on tapes.
 
 **This guide explains how to set up storage on tapes with Cold Archive.**
 
-> [!primary]
->
-> During the the Alpha period, tapes are not used.
-
 ## Requirements
 
 - [Getting started with AWS CLI](https://docs.ovh.com/fr/storage/s3/debuter-avec-s3/#utilisation-de-aws-cli)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.fr-fr.md]:
```suggestion
- [Getting started with AWS CLI](https://docs.ovh.com/fr/storage/object-storage/s3/getting-started-with-object-storage/#utilisation-de-aws-cli)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.it-it.md [hunk 17]
```diff
@@ -1,14 +1,12 @@
 ---
-title: Cold Archive - Getting started with Cold Archive (Alpha) (EN)
+title: Cold Archive - Getting started with Cold Archive (Beta)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.it-it.md]:
```suggestion
title: Cold Archive - Getting started with Cold Archive (Beta) (EN)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.it-it.md [hunk 18]
```diff
@@ -18,13 +16,9 @@ Restoration can take some time as it needs to be read on tapes.
...
 **This guide explains how to set up storage on tapes with Cold Archive.**
 
-> [!primary]
->
-> During the the Alpha period, tapes are not used.
-
 ## Requirements
 
-- [Getting started with AWS CLI](https://docs.ovh.com/it/storage/s3/getting-started-with-s3/#using-the-aws-cli)
+- [Getting started with AWS CLI](https://docs.ovh.com/gb/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.it-it.md]:
```suggestion
- [Getting started with AWS CLI](https://docs.ovh.com/it/storage/object-storage/s3/getting-started-with-object-storage/#using-aws-cli)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.pl-pl.md [hunk 19]
```diff
@@ -1,14 +1,12 @@
 ---
-title: Cold Archive - Getting started with Cold Archive (Alpha) (EN)
+title: Cold Archive - Getting started with Cold Archive (Beta)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.pl-pl.md]:
```suggestion
title: Cold Archive - Getting started with Cold Archive (Beta) (EN)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.pl-pl.md [hunk 20]
```diff
@@ -18,13 +16,9 @@ Restoration can take some time as it needs to be read on tapes.
...
 **This guide explains how to set up storage on tapes with Cold Archive.**
 
-> [!primary]
->
-> During the the Alpha period, tapes are not used.
-
 ## Requirements
 
-- [Getting started with AWS CLI](https://docs.ovh.com/pl/storage/s3/getting-started-with-s3/#using-the-aws-cli)
+- [Getting started with AWS CLI](https://docs.ovh.com/gb/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.pl-pl.md]:
```suggestion
- [Getting started with AWS CLI](https://docs.ovh.com/pl/storage/object-storage/s3/getting-started-with-object-storage/#using-aws-cli)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.pt-pt.md [hunk 21]
```diff
@@ -1,14 +1,12 @@
 ---
-title: Cold Archive - Getting started with Cold Archive (Alpha) (EN)
+title: Cold Archive - Getting started with Cold Archive (Beta)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.pt-pt.md]:
```suggestion
title: Cold Archive - Getting started with Cold Archive (Beta) (EN)
```

File: pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.pt-pt.md [hunk 22]
```diff
@@ -18,13 +16,10 @@ Restoration can take some time as it needs to be read on tapes.
...
 **This guide explains how to set up storage on tapes with Cold Archive.**
 
-> [!primary]
->
-> During the the Alpha period, tapes are not used.
-
 ## Requirements
 
-- [Getting started with AWS CLI](https://docs.ovh.com/pt/storage/s3/getting-started-with-s3/#using-the-aws-cli)
+- [Getting started with AWS CLI](https://docs.ovh.com/gb/en/storage/s3/getting-started-with-s3/#using-the-aws-cli)
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.pt-pt.md]:
```suggestion
- [Getting started with AWS CLI](https://docs.ovh.com/pt/storage/object-storage/s3/getting-started-with-object-storage/#using-aws-cli)
```

**Review by OVH-Team-Guides — CHANGES_REQUESTED** [review]
Please re-integrate the updated canonical link on the following subs: de-de, es-es, es-us, it-it, pl-pl, pt-pt.
Please also apply links changes.

**Review by OVH-Team-Guides — APPROVED** [review]

**Review by Pofilo — APPROVED** [review]

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @jlutran

**Pull request merged**